src/api/procedures/utils.ts

```diff
@@ -533,6 +536,20 @@ export async function assertAddRelayerPayingKeyAuthorizationValid(
   }
 }
 
+/**
+ * @hidden
+ *
+ * Asserts a target is a Account
+ */
+function assertIsSigner(target: Identity | Account): asserts target is Account {
```

Comment:
Shouldn't it be `assertIsAccount`? Also, you can use the `Signer` type instead of `Identity | Account`